docs/source/tutorials/one_class_segmentation_2d_unet.rst

```diff
@@ -222,23 +448,33 @@ Evaluate model
 
     To test the trained model on the testing sub-dataset and compute evaluation metrics, run:
 
-    .. code-block:: bash
+    .. tabs::
+
+        .. tab:: Command Line Interface
+
+            .. code-block:: bash
 
-       ivadomed --test -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
+               ivadomed --test -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
 
-    If you prefer to use config files over CLI flags, set ``command`` to the following in you config file:
+        .. tab:: JSON File
 
-    .. code-block:: json
+            If you prefer to use config files over CLI flags, set ``command`` to the following in you config file:
 
-       "command": "test"
+            .. code-block:: json
 
-    You can also set ``path_output``, and ``path_data`` arguments in your config file.
+               "command": "test"
+
+            You can also set ``path_output``, and ``path_data`` arguments in your config file, usually around **Line Number 2, 4, and 12** in the `config.json` respectively.
 
     Then run:
 
-    .. code-block:: bash
+    .. tabs::
+
+        .. tab:: Command Line Interface
+
+            .. code-block:: bash
 
-       ivadomed -c config.json
+                ivadomed -c config.json
```

Comment:
the "Then run" part only applies if the "test" command is set in the JSON tab above, which is not clear here.

Reply:
Great point, it was an indentation issue, this should have been under the "JSON File" block and not gets elevated in level.